src/dotnet/Plugin.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project>
 
   <PropertyGroup>
-    <SdkVersion>2021.3.1</SdkVersion>
+    <SdkVersion>2021.4.0</SdkVersion>
```

Comment:
While not ideal, major and minor are dictated by the R#/Rider version – at least for the plugins that I maintain. This is to reflect what version it belongs to.